### PR TITLE
Copyedit

### PR DESCRIPTION
**Pull request by Artoria2e5**

### CODE REVIEW

File: themes/nohttps/layout/index.jade [hunk 1]
```diff
@@ -94,22 +94,45 @@ html
...
         :markdown
-          SSL ensures both that data is not tapped and not tampered with, but SSL also make the request cannot be cached by proxy nodes in the network. Caching and proxies are very important parts of the HTTP protocol, but if you use HTTPS, the proxy nodes cannot read HTTP headers even.
+          SSL ensures the integrity and privacy of data transmissions. By stopping others from looking at your resources, SSL also stops the request from being cached by proxy nodes in the network. Caching and proxies are very important parts of the HTTP protocol, but proxy nodes cannot even read HTTP headers if you use HTTPS.
           
-          Use HTTPS for requests that contain user privacy is fine, but more requests are pictures, style sheets and scripts. **These requests only require tamper-proofing, and do not require tapped-proofing**, so there is no need to use SSL.
+          //- more -> most?
+          It's fine to use HTTPS for requests that contain private info. However, a bigger part of everyday requests are actually pictures, stylesheets and scripts. **These requests only require tamper-proofing**, and there is no need to use SSL's eavesdropping protection.
```
Comment (Artoria2e5) [inline themes/nohttps/layout/index.jade:101]: Not proposal; TR. Too lazy to change both zh and en.

File: themes/nohttps/layout/index.jade [hunk 2]
```diff
@@ -94,22 +94,45 @@ html
...
+          The solution is, in fact, very similar to what happens in many Linux distributions' package repositories: the package itself has a separate GPG signature to prevent modificaton, so the package file can be transmitted over HTTP, allowing it to be proxied and cached. However, there is a trend for mirrors to add HTTPS support, creating a waste of resources, and increasing latency perceptibly.
           
-          Of course, I know the most part of reason is ISP does not follow the rules to caching the resources, but I think use HTTPS in anywhere is not what we want to be, so I built this site to find a better solution.
-      
-          ## Who signed the manifesto
+          Yes, I acknowledge that many of you are deploying HTTPS is due to ISPs' unruly caching causing problems. But I believe putting HTTPS everywhere as a workaround is not what we want to do, so I built this site to explore better possibilities.
```
Comment (Artoria2e5) [inline themes/nohttps/layout/index.jade:112]: \* No pun intended. I personally use HTTPS Everywhere, and surf HTTPS everywhere.

**Comment by Artoria2e5** [comment]
Found some fun reading on SRI/mixed-content: https://lists.w3.org/Archives/Public/public-webappsec/2014Nov/0273.html

**Comment by Solo1st** [comment]
mark

**Comment by infinnie** [comment]
mark

File: themes/nohttps/layout/index.jade [hunk 3]
```diff
@@ -94,22 +94,45 @@ html
...
+          
+          This is why we are writing this manifesto – we want to change the status quo. Full-site HTTPS is not the only solution to resource integrity.
+          
+          > What happens when the checksums don't match?
+          
+          The browser will refuse to load the resource, just like how it should react to an HTTPS MITM attack.
+          
+          > Why are you so concerned with resources wasted by HTTPS?
+          
+          Despite a continuous increase in computing power, we are now hitting physical limits on network latency, which can be greatly reduced by serving resources from nearby caching nodes.
```
Comment (jysperm) [inline themes/nohttps/layout/index.jade]: 这句想表达的是「虽然计算力在增长，**但** 网络延迟已经到了极限」，感觉要加一个 But.

**Review by jysperm — APPROVED** [review]
非常感谢，学到了很多，有些逻辑比中文版还要顺畅。

**Comment by jysperm** [comment]
Thanks again!

**Pull request merged**